free45/toplevels/bsg_ac_black_parrot_single_core_rc3/tcl/dc_init/dc.mcmm.scenarios.tcl

```diff
@@ -0,0 +1,252 @@
+#################################################################################
```

Comment:
All of these folders in tcl/dc_init can be deleted. They're not used in this flow

Reply:
Should I get rid of the whole folder `dc_init/*`? What about `dc_spg/*`?

Reply:
dc_init, dc_spg and icc are all used only in the 40nm flow